### ISSUE TITLE
Riak 1.3.0 tarball is broken

### ISSUE THREAD

**Issue by kientzlecatch** [basho/riak#295]
I originally filed this as a documentation issue but it was suggested I file it here instead:
  https://github.com/basho/basho_docs/issues/265

The instructions at docs.basho.com (http://docs.basho.com/riak/latest/tutorials/fast-track/Building-a-Development-Environment/ and various pages linked from there) suggest downloading a tarball and then running make.

Unfortunately, the makefile immediately tries to do git operations; the tarball doesn't have a .git directory, is therefore not a git checkout, and those operations immediately fail.   (This is different from the "building without Internet connectivity" issue described on some of those pages.)

I've gotten further by following the instructions for working from a github clone (but encountered breakage there as well; I'll report those issues separately.)

**Comment by jaredmorrow** [comment]
I just disconnected my machine from the network and built from the 1.3.0 tarball, I hit the leveldb issue as described in the "building without internet connectivity", in the 'make' step.  Once I put the leveldb directory in place everything built fine without any network connectivity.

I see where you hit your problem though.  In the 'make devrel' step it did try to go out to the network to re-clone all of the dependencies.  To fix this, simply change this line in the Makefile: https://github.com/basho/riak/blob/riak-1.3.0/Makefile#L79 from

```
dev% : all riaknostic
```

to 

```
dev% : riaknostic
```

Remove the 'dev' directory to make sure everything is clean and try to run 'make devrel' again.  

Let me know how that goes.

**Comment by kientzlecatch** [comment]
For the record:  I have full Internet connectivity on my dev machine.  The "building without internet connectivity" instructions do not describe the problems I'm having.

Now I see where I went wrong.  The full messages I got before were:

$ make rel
fatal: Not a git repository (or any of the parent directories): .git
./rebar get-deps
env: escript: No such file or directory

The git "fatal" error confused me.  The real problem was not having the freshly-installed Erlang in my path.  (Since I couldn't use MacPorts because of the Erlang version problem, I had to use 'kerl' to install Erlang and kerl does not set up your path for you you.)

After fixing that, the instructions work better.  There's still the snappy build failure on Mac to work around, but I've already puzzled that one out.
